plugins/kubernetes/app/models/watchers/deploy_watcher.rb

```diff
@@ -0,0 +1,83 @@
+require 'celluloid'
+require 'celluloid/autostart'
```

Comment:
celluloid/autostart already includes celluloid. I would however go for celluloid/current to force ourselves to use the new APIs. Celluloid is on its way to version 1.0 and there are some changes coming, which are better addressed now, see https://github.com/celluloid/celluloid/wiki/DEPRECATION-WARNING